src/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY GhDampingFunctions)
```

Comment:
What about the name `GhConstraintDamping`? And naming the directory `ConstraintDamping` (the `Gh` part is clear because of the subdirectory nature, I think)